ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,271 @@
+    "<br>Last Verified to Run: **N/A -- in development**\n",
+    "\n",
+    "This project addresses issue [#63: HSC Re-run](https://github.com/LSSTScienceCollaborations/StackClub/issues/63)\n",
+    "\n",
+    "This notebook demonstrates the [LSST Science Piplines data processing tutorial](https://pipelines.lsst.io/) with emphasis on how a given [obs package](https://github.com/lsst/obs_base) works under the hood of the command line tasks. It makes use of the [obs_subaru](https://github.com/lsst/obs_subaru) package to measure a forced photometry light curve for a small patch in the HSC sky in the [ci_hsc](https://github.com/lsst/ci_hsc/) repository. \n",
+    "\n",
+    "### Learning Objectives:\n",
+    "After working through and studying this notebook you should be able to understand how to use the DRP pipeline from image visualization through to a forced photometry light curve. Specific learning objectives include: \n",
+    "   1. [configure command-line tasks](https://pipelines.lsst.io/v/w-2018-12/modules/lsst.pipe.base/command-line-task-config-howto.html) for your science case\n",
+    "   2. TODO\n",
+    "   \n",
+    "Other techniques that are demonstrated, but not empasized, in this notebook are\n",
+    "   1. Use the `butler` to fetch data\n",
+    "   2. Visualize data with the LSST Stack\n",
+    "   3. TODO\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "\n",
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import eups.setupcmd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I: Interacting with data. Introduction to the Butler\n",
+    "https://pipelines.lsst.io/getting-started/data-setup.html\n",
+    "\n",
+    "Part I runs the following command-line tasks\n",
+    "```\n",
+    "eups list lsst_distrib\n",
+    "setup -j -r /home/jmyles/repositories/ci_hsc\n",
+    "echo \"lsst.obs.hsc.HscMapper\" > /home/jmyles/DATA/_mapper\n",
+    "ingestImages.py /home/jmyles/DATA /home/jmyles/repositories/ci_hsc/raw/*.fits --mode=link\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/CALIB/ /home/jmyles/DATA/CALIB\n",
+    "installTransmissionCurves.py /home/jmyles/DATA\n",
+    "mkdir -p /home/jmyles/DATA/ref_cats\n",
+    "ln -s /home/jmyles/repositories/ci_hsc/ps1_pv3_3pi_20170110 /home/jmyles/DATA/ref_cats/ps1_pv3_3pi_20170110\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!eups list lsst_distrib\n",
+    "datarepo = \"/home/jmyles/repositories/ci_hsc/\"\n",
```

Comment:
To make this runnable by anyone, we'll need to use $USER instead of jtmykes, and `mkdir -p` to make a new director to use.